.github/workflows/data_quality_outputs.yml

```diff
@@ -0,0 +1,34 @@
+#
+#  1. Check we get expected outputs from data quality parameters on sample data
+#
+name: data_quality_outputs
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  parsing:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2 
+
+      - name: Compile Parser
+        run: javac -cp java/JTransforms-3.1-with-dependencies.jar java/*.java
+        
+      
+      - name: Load data
+        run: bash utilities/downloadDataModels.sh
+       
+      - name: Summarise sample
+        run: |
+            import accelerometer
```

Comment:
Don't think you need to do `import accelerometer` but setting up the Python environment will do because the daemon running the tests will not have python or anything packages that we use by default. https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-python could be of help :D